src/FixedDataTableCell.js

```diff
@@ -193,6 +194,8 @@ class FixedDataTableCell extends React.Component {
       newState.displacement = 0;
     }
 
+    this.DIR_SIGN = props.isRTL ? -1 : 1;
```

Comment:
let's remove `this.DIR_SIGN` and just make use of `props.isRTL` in the render method